scripts/export_environment.py

```diff
@@ -276,7 +276,8 @@ def export_env(env: Environment, output: Optional[TextIO]) -> None:
 
 
 def redact(k: str, v: str) -> str:
-    return 'REDACTED' if any(s in k.lower() for s in ('secret', 'password', 'token')) else v
+    secret_affixes = ('secret', 'password', 'token', 'key')
```

Comment:
`key` is too common. It may or may not be a secret. And these aren't "affixes", these are substrings, which makes `key` even less appropriate.

Reply:
Have any keys been compromised by the fact that, before this chanhe, AZUL_VELOCITY_ZENHUB_KEY and AZUL_VELOCITY_ZENHUB_PKEY were not redacted?

Reply:
No, this change is just for consistency. Should I drop this commit entirely?

Reply:
If this is the best you can do, then yes, as currently coded, this change does more harm than good.

Reply:
Okay. Given that:
* your [comment](https://github.com/DataBiosphere/azul/pull/2450#discussion_r523271800) implies that this is not a priority,
* the only two instances where these keys are even stored are on my machine and in this repository's GitHub Actions configuration,
* the current behavior, without my changes, may already be too broad (for example, `AZUL_CART_EXPORT_MIN_ACCESS_TOKEN_TTL` is redacted and may not need to be),

I'll drop this commit before the next review.

Reply:
Sounds good. I might take a shot at this once I review it. Can't be too hard to get right.